core/include/moveit/task_constructor/solvers/pipeline_planner.h

```diff
@@ -101,7 +104,9 @@ class PipelinePlanner : public PlannerInterface
 	void setSolutionSelectionFunction(
 	    const moveit::planning_pipeline_interfaces::SolutionSelectionFunction& solution_selection_function);
 
-	/** \brief If not yet done, initialize pipelines from pipeline_id_planner_id_map
+	/** \brief This function is called when an MTC task that uses this solver is initialized. If no pipelines are
```

Comment:
Not a strong opinion here, but I think more verbose comments are helpful. Where some of them wrong?

Reply:
Which information is missing with the shortened version?
\brief comments should be brief one-liners :wink: